azurelinuxagent/common/event.py

```diff
@@ -127,6 +127,7 @@ class WALAEventOperation:
     Update = "Update"
     VmSettings = "VmSettings"
     VmSettingsSummary = "VmSettingsSummary"
+    FeatureFlag = "FeatureFlag"
```

Comment:
"FeatureFlag" is too generic, can we use a different name?

Reply:
If it is meant for multiple features I think the message should be json or something like that so that we can parse it for individual features. At this point it is used only for agent versioning. If you want to use it for all features, you should also use it for multi-config and fastrack.

I think you should make this one specific to versioning